CONTRIBUTING.md

```diff
@@ -111,6 +111,38 @@ To generate this data, please run `./scripts/regenerate_test_rustdocs.sh`.
 To use a specific toolchain, like beta or nightly, pass it as
 an argument: `./scripts/regenerate_test_rustdocs.sh +nightly`.
 
+## What are those `.snap` or `.snap.new` files generated via `cargo test` 
```

Comment:
Does it make sense to make CI fail if `.snap.new` files are checked into the repo? Will it do that already?

Reply:
It will somewhat already do that (unless one creates a `.snap.new` and the same `.snap` file, will have to test that part)

Reply:
Yes, having both a `.snap.new` and a `.snap` file was what I would love to make sure we prevent. Because if that happens, it will pollute the repo and confuse the local setups of everyone else, since everyone's insta may think there are altered snapshots.